### PR TITLE
chore: tdp related questions

### PR DESCRIPTION
**Pull request by yuqizhou77**
- updated questions for project creation from Dev Portal
    - if app with tab feature
       - ask users whether they want to update website url to localhost for debugging 
       image

       - ask users whether they want to update content url to localhost for debugging 
         image

   - if app with bot feature
      - create a new bot for debugging
      image

  - if app with message extension feature
     - create a new bot for debugging
      image

   - bot and message extension
      - create a new bot for debugging
      image

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -719,13 +719,17 @@
...
   "core.HostTypeOptionSPFx.label": "SharePoint Framework (SPFx)",
-  "core.updateBotIdsQuestion.title": "We will create a new bot for you to get started with Teams Toolkit",
+  "core.updateBotIdsQuestion.title": "Create new bot(s) for debugging",
   "core.updateBotIdsQuestion.placeholder": "Unselect to keep with the original one.",
-  "core.updateContentUrlQuestion.title": "We will update the content url of following tab(s) for you to get started with Teams Toolkit",
-  "core.updateWebsiteUrlQuestion.title": "We will update the website url of following tab(s) for you to get started with Teams Toolkit",
-  "core.updateContentUrlOption.description": "update the content url from %s to %s",
-  "core.updateWebsiteUrlOption.description": "update the website url from %s to %s",
-  "core.updateUrlQuestion.placeholder": "Unselect to keep with the original url",
+  "core.updateBotIdForBot.description": "update botId %s to \"{{BOT_ID}}\" in manifest.json",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:724]: to be consistent, to capitalize the first character: u -> U

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -719,13 +719,17 @@
...
+  "core.updateBotIdForMessageExtension.description": "update botId %s to \"{{BOT_ID}}\" in manifest.json",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:725]: same as line above: capitalize u

File: packages/fx-core/resource/package.nls.json [hunk 3]
```diff
@@ -719,13 +719,17 @@
...
+  "core.updateBotIdForBot.label": "Bot",
+  "core.updateBotIdForMessageExtension.label": "Message extension",
+  "core.updateContentUrlQuestion.title": "Configure content URL(s) for debugging",
+  "core.updateWebsiteUrlQuestion.title": "Configure website URL(s) for debugging",
+  "core.updateContentUrlOption.description": "update the content URL from %s to %s",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:730]: capitalize u?

File: packages/fx-core/resource/package.nls.json [hunk 4]
```diff
@@ -719,13 +719,17 @@
...
+  "core.updateWebsiteUrlOption.description": "update the website URL from %s to %s",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:731]: capitalize u?

File: packages/fx-core/resource/package.nls.json [hunk 5]
```diff
@@ -719,13 +719,17 @@
   "core.SampleSelect.buttons.viewSamples": "View samples",
   "core.HostTypeOptionAzure.label": "Azure",
   "core.HostTypeOptionSPFx.label": "SharePoint Framework (SPFx)",
-  "core.updateBotIdsQuestion.title": "We will create a new bot for you to get started with Teams Toolkit",
+  "core.updateBotIdsQuestion.title": "Create new bot(s) for debugging",
   "core.updateBotIdsQuestion.placeholder": "Unselect to keep with the original one.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:723]: pls change 'one' to 'BotID' or any more specific term.

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (yuqizhou77) [reply packages/fx-core/resource/package.nls.json]: updated strings. Please see the attachments above :)

File: packages/fx-core/resource/package.nls.json [hunk 6]
```diff
@@ -719,13 +719,20 @@
...
-  "core.updateContentUrlQuestion.title": "We will update the content url of following tab(s) for you to get started with Teams Toolkit",
-  "core.updateWebsiteUrlQuestion.title": "We will update the website url of following tab(s) for you to get started with Teams Toolkit",
-  "core.updateContentUrlOption.description": "update the content url from %s to %s",
-  "core.updateWebsiteUrlOption.description": "update the website url from %s to %s",
-  "core.updateUrlQuestion.placeholder": "Unselect to keep with the original url",
+  "core.updateBotIdsQuestion.title": "Create new bot(s) for debugging",
+  "core.updateBotIdsQuestion.placeholder": "Unselect to keep with the original value of botId",
+  "_core.updateBotIdsQuestion.placeholder.comment": "'botId' is the field name which should not be localized.",
+  "core.updateBotIdForBot.description": "Update botId %s to \"${{BOT_ID}}\" in manifest.json",
+  "_core.updateBotIdForBot.description.comment": "'botId' and '${{BOT_ID}}' should not be localized. 'manifest.json' is the file name which should also not be localized.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:726]: "which should also not be localized." -> "which should not be localized" to make it straightforward.

File: packages/fx-core/resource/package.nls.json [hunk 7]
```diff
@@ -719,13 +719,20 @@
...
+  "core.updateBotIdForMessageExtension.description": "Update botId %s to \"${{BOT_ID}}\" in manifest.json",
+  "_core.updateBotIdForMessageExtension.description.comment": "'botId' and '${{BOT_ID}}' should not be localized. 'manifest.json' is the file name which should also not be localized.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:728]: "should also not be localized." -> "should not be localized"

**Review by sffamily — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**